src/components/image-uploader/ImageUploader.jsx

```diff
@@ -144,10 +155,18 @@ export default class ImageUploader extends React.Component {
   }
 
   handleDrop(files) {
+    const { acceptType, checkImageType } = this.props;
```

Comment:
do you need two props here ? you could just use acceptType with the default of any, and run the typecheck every time you upload

Reply:
yeah, could omit `checkImageType` from props, with added variable similar to this to properly cover `image/*` type case